### PR TITLE
Includes the CRL Distribution Points option

### PR DESCRIPTION
**Pull request by srbraun**
EDIT:

Documentation for https://github.com/jetstack/cert-manager/pull/2625

### CODE REVIEW

File: content/en/docs/configuration/ca.md [hunk 1]
```diff
@@ -55,7 +55,15 @@ spec:
   ca:
     secretName: ca-key-pair
 ```
-
+Optionally, you can specify (CRL)[https://en.wikipedia.org/wiki/Certificate_revocation_list] Distribution Points. An array of `Strings` each of which identifies the location of the CRL from which the revocation of this certificate can be checked:
```
Comment (meyskens) [inline content/en/docs/configuration/ca.md]: Any particular reason we backtick Strings? I think just `strings` (with not backticks) will do in this case

Comment (srbraun) [reply content/en/docs/configuration/ca.md]: done

**Comment by munnerz** [comment]
Thanks for putting this feature together! 😄 

/lgtm
/approve

**Comment by srbraun** [comment]
> Thanks for putting this feature together! smile

Thank you @munnerz  for the support.

**Pull request merged**